src/test/java/org/fogbowcloud/manager/core/processors/SpawningProcessorTest.java

```diff
@@ -60,243 +55,279 @@ public void setUp() {
-        
+
         this.thread = null;
     }
-    
+
     @After
     public void tearDown() {
         if (this.thread != null) {
             this.thread.interrupt();
         }
         super.tearDown();
     }
-    
+
+    // test case: In calling the processSpawningOrder() method for any order other than spawning,
+    // you must not make state transition by keeping the order in your source list.
     @Test
-    public void testTryingProcessComputeOrderNotSpawning() throws Exception {
+    public void testProcessComputeOrderNotSpawning() throws Exception {
+
+        // set up
         Order order = spyComputeOrder();
         order.setOrderState(OrderState.OPEN);
         this.openOrderList.addItem(order);
-        
+
+        // exercise
         this.spawningProcessor.processSpawningOrder(order);
 
-        assertNotNull(this.openOrderList.getNext());
-        assertNull(this.fulfilledOrderList.getNext());
+        // verify
+        Assert.assertNotNull(this.openOrderList.getNext());
+        Assert.assertNull(this.fulfilledOrderList.getNext());
     }
-    
+
+    // test case: When running thread in SpawningProcessor, if there is no SSH connectivity as the
+    // reverse tunnel, the method processSpawningOrder() must not change OrderState to Fulfilled and
+    // must remain in Spawning list.
     @SuppressWarnings("static-access")
     @Test
     public void testRunProcessComputeOrderWithoutSSHConnectivity() throws Exception {
+
+        // set up
         Order order = spyComputeOrder();
         order.setOrderState(OrderState.SPAWNING);
         this.spawningOrderList.addItem(order);
-        
-        Instance orderInstance = spy(new ComputeInstance(FAKE_INSTANCE_ID));
+
+        Instance orderInstance = Mockito.spy(new ComputeInstance(FAKE_INSTANCE_ID));
         orderInstance.setState(InstanceState.READY);
         order.setInstanceId(FAKE_INSTANCE_ID);
 
-        doReturn(orderInstance).when(this.cloudConnector).getInstance(any(Order.class));
+        // exercise
+        Mockito.doReturn(orderInstance).when(this.cloudConnector)
+                .getInstance(Mockito.any(Order.class));
+        Mockito.when(this.tunnelingService.getExternalServiceAddresses(Mockito.anyString()))
+                .thenReturn(new HashMap<>());
+        Mockito.when(this.sshConnectivity
+                .checkSSHConnectivity(Mockito.any(SshTunnelConnectionData.class)))
+                .thenReturn(false);
 
-        when(this.tunnelingService.getExternalServiceAddresses(anyString())).thenReturn(new HashMap<>());
-        
-        when(this.sshConnectivity.checkSSHConnectivity(any(SshTunnelConnectionData.class))).thenReturn(false);
-        
         this.thread = new Thread(this.spawningProcessor);
         this.thread.start();
         this.thread.sleep(500);
 
-        assertNotNull(this.spawningOrderList.getNext());
-        assertNull(this.fulfilledOrderList.getNext());
+        // verify
```

Comment:
same comment as above

Reply:
... I mean, it is better to check the element in the list is equals to the expected one.